geom/type_sum.go

```diff
@@ -584,19 +584,29 @@ func (g Geometry) Centroid() (Point, bool) {
 }
 
 // Area gives the area of the Polygon or MultiPolygon. If the Geometry is not a
```

Comment:
Nit: Comment could be updated to indicate that it also gives the area of GeometryCollections.